src/test/java/picard/illumina/IlluminaLaneMetricsCollectorTest.java

```diff
@@ -50,14 +50,13 @@ public void testWriteLaneMetrics(final String testRun) throws Exception {
     }
 
     @Test(dataProvider = "testCollectIlluminaLaneMetrics")
-    public void testCollectIlluminaLaneMetrics(final String testRun, final ReadStructure readStructure, final boolean isNovaSeq) throws Exception {
+    public void testCollectIlluminaLaneMetrics(final String testRun, final ReadStructure readStructure) throws Exception {
         for (final boolean useReadStructure : Arrays.asList(true, false)) {
             final File runDirectory = new File(TILE_RUN_DIRECTORY, testRun);
             final CollectIlluminaLaneMetrics clp = new CollectIlluminaLaneMetrics();
             clp.OUTPUT_DIRECTORY = IOUtil.createTempDir("illuminaLaneMetricsCollectorTest", null);
             clp.RUN_DIRECTORY = runDirectory;
             clp.OUTPUT_PREFIX = "test";
-            clp.IS_NOVASEQ = isNovaSeq;
             if (useReadStructure) clp.READ_STRUCTURE = readStructure;
             clp.doWork();
```

Comment:
Any concern that the metrics generated would be different if the parser misidentified the file as V2 vs V3?

Reply:
If they are misidentified then the parsing with throw an error. They are significantly different in structure.